Automattic/wp-calypso
# Use New Common Component in New Domain Flow

#### Changes proposed in this Pull Request

* Use New Common G Suite New User List Component in New Domain Flow

#### Testing instructions

* Begin adding a new domain to existing site
* Get to G Suite Upsell
* Verify new user list behavior is correct
* Verify all users are added to card correctly
